test/parallel/test-http-should-keep-alive.js

```diff
@@ -41,34 +42,27 @@ const SHOULD_KEEP_ALIVE = [
   true,  // HTTP/1.1, Connection: keep-alive
   false  // HTTP/1.1, Connection: close
 ];
-let requests = 0;
-let responses = 0;
 http.globalAgent.maxSockets = 5;
 
+const countdown = new Countdown(SHOULD_KEEP_ALIVE.length , () => server.close());
+
+const getCountdownIndex = () => SERVER_RESPONSES.length - countdown.remaining
```

Comment:
It seems like there's still no semi-colon at the end of this line btw.

Reply:
Hi, I`m probably not running the eslint as should. the command 
just pass without no issues.
how can I make sure its ok?

Reply:
You could try running `make lint-js` from your command line, that should lint the files. The `-require` might not be caught but the missing semi-colon definitely should.